Remove ssh commands and add cloud-config in USER_DATA

nicolasvienot opened the pull request:
closes #149 

This is the first draft for the removal of the ssh commands.
See #149

It aims to :
- Simplify the whole process
- Have the same structure in all the meilisearch-cloud repositories
- Easier implementation of the future tests/CI
- Make the maintenance easier

What has been done :
- Remove the ssh commands launched after the droplet creation
- Remove the SSH_KEYS_FINGERPRINTS
- cloud-config.yaml is now unique for each repository (not in cloud-scripts anymore) and directly put in USER_DATA

To do :
- Test
- Update contributing
- @eskombro would you prefer the cloud-config.yaml to be an independant file instead of a variable USER_DATA in config.py?

nicolasvienot committed:
Remove ssh commands and add cloud-config in USER_DATA

nicolasvienot committed:
Remove commands in build_image.py

eskombro reviewed (commented):
- I would like to rewrite the UFW commands (we can do better :) )
- We don't need the `rm -rf /var/log/*` IMO since the clean script should delete the logs
- How are we checking the output of `img_check.sh` dynamically? We can not just run teh script, because we won't be aware if there is a Warning/Error

nicolasvienot committed:
Use could-config.yaml from cloud-scripts

nicolasvienot committed:
Clean

nicolasvienot commented:
> I would like to rewrite the UFW commands (we can do better :) )

I added an issue about that, as we agreed to focus on the 'ssh removal'. That's on the cloud-scripts repositories (It's common for all the cloud repositories, yay 🎉) -> [here](https://github.com/meilisearch/cloud-providers/issues/41)

> We don't need the rm -rf /var/log/* IMO since the clean script should delete the logs

Done in meilisearch/cloud-scripts 🎉

> How are we checking the output of img_check.sh dynamically? We can not just run teh script, because we won't be aware if there is a Warning/Error

As seen with you, I added an issue about that as well in the meilisearch-digitalocean #153

nicolasvienot commented:
Update :
- The cloud-config.yaml is now back in the meilisearch/cloud-scripts
- Each provider has its own cloud-config.yaml located in cloud-scripts/scripts/providers
